### ISSUE TITLE
Relocate community page

### ISSUE THREAD

**Issue by jcfr** [Slicer/Slicer#5283]
## Is your feature request related to a problem? Please describe.

Provide a homogenous user experience relocating community page from wiki to either readthedocs or new slicer.org website. 

## Describe the solution you'd like

Transition community wiki page to the new slicer.org website

## Describe alternatives you've considered

NA

## Additional context

See https://github.com/Slicer/Slicer/pull/5282

**Comment by lassoan** [comment]
@pieper to talk to Ron and Marianna. Useful to have on the same site for SEO.

**Comment by jcfr** [comment]
There are two parts:

### Move download and publication stats to the website. 

Approach discussed at https://github.com/Slicer/slicer.org/issues/63#issuecomment-677756609

### Move history of publications to the website

Publication for each years would  be organized in a markdown like document

**Comment by jamesobutler** [comment]
@pieper Did you hear from the others about this?

**Comment by pieper** [comment]
Yes, moving the community page into the slicer.org website repo makes sense and the person who had been maintaining the wiki can switch over to updating markdown and making pull requests.  The next step would be for someone to migrate the existing content into markdown with a script of some kind.  The existing data is consistently formatted, but it's wikitext not a database.
